tools/Unix.mk

```diff
@@ -589,36 +579,43 @@ pass2dep: context tools/mkdeps$(HOSTEXEEXT) tools/cnvwindeps$(HOSTEXEEXT)
 
 config:
 	$(Q) $(MAKE) clean_context
+	$(Q) $(MAKE) context
 	$(Q) $(MAKE) apps_preconfig
 	$(Q) APPSDIR=${CONFIG_APPS_DIR} EXTERNALDIR=$(EXTERNALDIR) kconfig-conf Kconfig
 
 oldconfig:
 	$(Q) $(MAKE) clean_context
+	$(Q) $(MAKE) context
 	$(Q) $(MAKE) apps_preconfig
 	$(Q) APPSDIR=${CONFIG_APPS_DIR} EXTERNALDIR=$(EXTERNALDIR) kconfig-conf --oldconfig Kconfig
 
 olddefconfig:
 	$(Q) $(MAKE) clean_context
+	$(Q) $(MAKE) context
 	$(Q) $(MAKE) apps_preconfig
 	$(Q) APPSDIR=${CONFIG_APPS_DIR} EXTERNALDIR=$(EXTERNALDIR) kconfig-conf --olddefconfig Kconfig
 
 menuconfig:
 	$(Q) $(MAKE) clean_context
+	$(Q) $(MAKE) context
```

Comment:
Why do we have different indentation here? (Same in other places)

Reply:
makefile rules are indented with tabs, and makefiles lists are intended with spaces

Reply:
Probably this is a bug of GitHub mobile app. The file view seems not having such display issue.